features/cellular/easy_cellular/CellularConnectionFSM.cpp

```diff
@@ -468,6 +423,18 @@ void CellularConnectionFSM::state_sim_pin()
     _cellularDevice->set_timeout(TIMEOUT_SIM_PIN);
     tr_info("Sim state (timeout %d ms)", TIMEOUT_SIM_PIN);
     if (open_sim()) {
+        bool success = false;
+        for (int type = 0; type < CellularNetwork::C_MAX; type++) {
+            if (!_network->set_registration_urc((CellularNetwork::RegistrationType)type, true)) {
+                success = true;
+            }
+        }
+        if (!success) {
+            tr_warning("Failed to set any URC's for registration");
```

Comment:
Please review the build logs. Looks like this is not defined (missing include ?) .